CHANGELOG.md

```diff
@@ -7,6 +7,188 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [16.0.0] - 2023-08-XX
+
+### Breaking changes
+
```

Comment:
Need to talk about higher level changes first:
- New concept of recipeUserId vs primary user ID you can link to docs here: supertokens.com/docs/thirdpartyemailpassword/user-object#primary-vs-recipe-user-id
- Introducing account linking functionality.